js/data/question.jsonc

```diff
@@ -0,0 +1,599 @@
+	"import_kcnav": "KCNavから読込",
+	
+	"ship_has_customised_stats": "ステータス編集有",
+	"ship_has_unknown_stats": "不明なステータスが含まれます",
+	
+	"load_options": "編成読み込み",
+	"from_text": "デッキビルダーから:",
+    "from_text_note": "本シミュ、デッキビルダー、KC3 ReplayのJSONファイルが読み込み可能です",
+	"load": "Load", // ok
+	"fleet": "艦隊",
+	"CTF": "空母機動部隊",
+	"STF": "水上打撃部隊",
+	"TCF": "輸送護衛部隊",
+	"from_kc3_file": ".kc3 ファイルから",
+	"from_preset": "プリセットから",
+	"clear_all": "艦隊リセット",
+	"export_deckbuilder": "デッキビルダー出力",
+	"stat_level": "レベル",
+	"stat_HP": "HP",
+	"stat_true_value_unknown": "このステータスの正確な値はまだ不明です。",
+    "stat_true_value_placeholder": "このステータスの正確な値はまだ不明であり、データが不足しています。手動での設定を推奨します。",
+    "stat_true_value_db_estimate": "このステータスの正確な値はまだ不明であり、データベースに送信されたデータに基づく推定値です。",
+	"stat_Firepower": "火力",
+	"stat_Torpedo": "雷装",
+	"stat_Anti-Air": "対空",
+	"stat_Armour": "装甲",
+	"stat_Evasion": "回避",
+	"stat_Anti-Submarine": "対潜",
+	"stat_Line of Sight": "索敵",
+	"stat_luck": "運",
+	"stat_range": "射程",
+	"stat_divebomb": "爆装",
+	"stat_accuracy": "命中",
+	"stat_torpedo_accuracy": "雷撃命中",
+    // bonus_dmg, bonus_acc, and bonus_eva should have a clean line break before 'ボーナス'
+    "bonus_dmg": "ダメージボーナス",
+	"bonus_dmg_desc": "マップ全体でダメージボーナスを付与します(倍率)",
+    "bonus_acc": "命中ボーナス",
+    "bonus_acc_desc": "マップ全体で命中ボーナスを付与します(倍率)",
+    "bonus_eva": "回避ボーナス",
+    "bonus_eva_desc": "マップ全体で回避ボーナスを付与します(倍率)",
+    // Who applies what to whom and how?
+	"debuff_dmg": "Debuff Dmg",
+	"debuff_dmg_desc": "Multiplier applied to LAST NODE FLAGSHIP only, stacks with above",
+	"slot_size": "搭載数",
+	"proficiency": "艦載機熟練度",
+	"improvement": "改修状況",
+	"hp_init": "HP 初期値",
+	"init": "開始HP",
+	"morale": "cond値",
+	"fuel_percent": "燃料残量(%)",
+    "ammo_percent": "弾薬残量(%)",
+	"air_sub_node": "対潜空襲マス",
+	"air_sub_support": "Support",// ?
+	"show_equipped_stats": "装備補正値を見る",
+	"autoscale_stats": "自動調整 {img_hp}/{img_ev}/{img_asw}/{img_los} with {img_lv}",
+	"preserve_bonus_on_change": "艦の変更時にボーナスを維持",
+	"show_additional_stats": "艦隊詳細",
+	"single_fleet": "通常艦隊",
+	"combined_fleet": "連合艦隊",
+	"carrier_task_force": "空母機動部隊",
+	"surface_task_force": "水上打撃部隊",
+	"transport_combined_fleet": "輸送護衛部隊",
+	"strike_force": "遊撃部隊",
+	"set_all_morale_to": "cond値一括適用",
+	
+	"fleet_air_power": "制空値",
+	"fleet_air_power_combined": "艦隊全体",
+	"fleet_air_power_lbas": "vs LBAS",
+	"transport": "TP(S)",
+	"effective_los": "Effective LoS (HQ120)",
+	"shell_power": "砲戦火力",
+	"nb_power": "夜戦火力",
+	"asw_power": "対潜",
+	"can_oasw": "先制対潜",
+	"standard_accuracy": "命中",
+	"fit_accuracy": "フィット砲命中",
+    "fit_accuracy_night": "フィット砲命中(夜戦)",
+	"standard_evasion": "回避",
+	"artillery_spot_rate": "特殊攻撃",
+    "in_ASP_AS": "制空: 確保 / 優勢",
+    "night_attack": "夜戦",
+    "anti_air": "対空",
+    "proportional": "割合撃墜",
+    "flat": "固定撃墜",
+    // The type of anti-aircraft cut-in (#1, #2...) is generally expressed as (1種, 2種...) in Japanese
+	"aaci": "対空CI",
+	"aaci_none": "不発",
+    "aaci_planes": "固定ボーナス",
+    "aaci_rate": "発動率",
+    // For what?

```

Comment:
Fleet-wide AACI activation rate. For example, if Ship1 and Ship2 both have 50% individually but Ship1 has priority, then Ship1 -> 50%, Ship2 -> 25%.